bashbot/bot.py

```diff
@@ -65,7 +65,10 @@ async def on_message(self, message: Message):
 
         if self.is_invoke(message):
```

Comment:
This if-condition is checking if message is starting with prefix + dot. So it is not possible to have a message starting with prefix + dot in elif branch.